src/model.rs

```diff
@@ -201,6 +202,21 @@ pub struct FSRS<B: Backend = NdArrayBackend> {
     device: B::Device,
 }
 
+#[wasm_bindgen]
+pub struct FSRSwasm(FSRS<NdArrayBackend>);
+
+#[wasm_bindgen]
+impl FSRSwasm {
+    pub fn next_interval(
+        &self,
+        stability: Option<f32>,
+        desired_retention: f32,
+        rating: u32,
+    ) -> u32 {
+        self.0.next_interval(stability, desired_retention, rating)
+    }
+}
```

Comment:
Using the unfortunately verbose technique described [here](https://users.rust-lang.org/t/how-to-generate-wasm-for-generic-types-for-a-given-specific-type/67164). I am _very_ new to Rust, so there could be something obvious I'm missing. In other functional languages I'd use currying... but not sure if that's possible/reasonable here.